compliance/get.go

```diff
@@ -69,7 +69,8 @@ func GetNH(c *fluent.GRIBIClient, wantACK fluent.ProgrammingResult, t testing.TB
 		fluent.NextHopEntry().
 			WithNetworkInstance(defaultNetworkInstanceName).
 			WithIndex(1).
-			WithIPAddress("192.0.2.3"))
+			WithIPAddress("192.0.2.3").
+			WithFIBProgrammed(wantACK))
```

Comment:
This will lead -- in some cases -- to having `WithFIBProgrammed(RIB_PROGRAMMED)` which seems erroneous right? i.e., we will never have a `fib_status` field return `RIB_PROGRAMMED` right?

Instead, should we either have `WithProgrammedStatus()` which could take arguments of FIB ACKed or RIB ACKed? Or alternatively, `WithFIBProgrammedStatus` which only allows for FIB ACK or failed?